Closure_Project/rest_api/huji_loader.py

```diff
@@ -0,0 +1,139 @@
+import io
+import os
+import zipfile
+from pathlib import Path
+from typing import Dict, Union, IO
+from tqdm import tqdm
+import tempfile
+import json
+
+import requests
+from django.db import transaction
+
+from Parser.CornerStoneParser import CORNER_STONE_ID_FILENAME, CORNER_STONE_ID_FILE_PATH
+from Parser.MoonParser import PARSED_TRACKS_FOLDER_NAME, PARSED_GROUPS_FOLDER_NAME
+from Parser.OfflineParser import TRACK_DUMP_FOLDER_PATH, GROUP_DUMP_FOLDER_PATH, COURSE_DUMP_FILE_PATH, CURRENT_DIR, \
+    COURSE_DUMP_FILENAME, PARSED_DATA_ZIP_PATH, PARSED_DATA_ZIP_URL
+from rest_api.models import Track, CourseGroup, Course
+
+
+def import_tracks(folder: str = str(TRACK_DUMP_FOLDER_PATH)) -> None:
+
+    track_file_paths = Path(folder).glob("*.json")
+    track_dicts = [load_json(str(path)) for path in track_file_paths]
+
+    existing_track_numbers = [t["track_number"] for t in track_dicts]
+    _, delete_dict = Track.objects.filter(track_number__in=existing_track_numbers).delete()
+    print(f"Deleted and re-created conflicting tracks, the resulting deletion counts: {delete_dict} ")
+
+    tracks = [Track(**dic) for dic in track_dicts]
+    Track.objects.bulk_create(tracks)
+
+
+def import_course_group(group_values: Dict) -> None:
+    track_id = group_values['track_id']
+    track = Track.objects.get(track_number=track_id)
+    del group_values['track_id']
+
+    group_values['track'] = track
+
+    course_ids = group_values['course_ids']
+    del group_values['course_ids']
```

Comment:
`course_ids = group_values.pop('course_ids')` would be prettier, but doesn't really matter